antho444/antho444/Ruby_Assignment.rb

```diff
@@ -0,0 +1,89 @@
+rooms = ["Bedroom", "Kitchen", "Blinds", "Livingroom" "Trash Can"]
+
+def dust(room)
+  puts "The #{room} has been cleaned"
+end
+
+def vacuum(room)
+  puts "The #{room} has been vacuumed."
+end
+
+def make_bed
+  puts "The bed has been made."
+end
+
+def mop_floor
+  puts "The floor has been mopped."
+end
+
+def wash_dishes
+  puts "The dishes have been washed."
+end
+
+def clean_toilet
+  puts "The toilet has been cleaned."
+end
+
+def clean_room(room)
+  if room == "bedroom"
+    vacuum(room)
+    make_bed
+  elsif room == "kitchen"
+    mop_floor
+    wash_dishes
+  elsif room == "bathroom"
+    mop_floor
+    clean_tub
+  else
+    puts "The #{room} requires no specific cleaning processes."
+  end
+
+  mop_floor if room == "kitchen" || room == "bathroom"
+
+  dust(room)
+  puts "The #{room} has been cleaned."
+end
+
+def print_robot_face
+  puts "__^_____^__"
+  puts "|         |"
+  puts "|  ^   ^  |"
+  puts "|    >    |"
+  puts "|    o    |"
+  puts "|_________|"
+end
+
+def print_robot_face_money
+  puts "__^_____^__"
+  puts "|         |"
+  puts "|  ^   ^  |"
+  puts "|    >    |"
+  puts "|    $    |"
+  puts "|_________|"
+end
+
+def clean_house(rooms)
+  puts "Thank you for procuring my service."
+  print_robot_face
+  puts "\n"
+  sleep(1)
+  rooms.each do |room|
+    puts "Begin cleaning #{room}..."
+    clean_room(room)
+    puts "*" * 40
+    puts "\n"
+    sleep(1)
+  end
+
+  puts "House cleaning complete. Please remit payment. Venmo preferred."
+  print_robot_face_money
+end
+
+
+puts "Which room would you like me to clean"
+puts "Bathroom, Bedroom, Blinds, Trash"
+a = gets.chomp
+def print__money
+  puts "$$$"
+
+end
```

Comment:
This seems to be unfinished - you've defined all the functions, but you aren't actually running clean_house anywhere.